packages/react-native-codegen/src/parsers/flow/components/props.js

```diff
@@ -94,6 +94,7 @@ function getTypeAnnotationForArray(name, typeAnnotation, defaultValue, types) {
         name: 'ImageSourcePrimitive',
       };
     case 'ColorValue':
+    case 'ProcessedColorValue':
```

Comment:
Can you add this to the test fixture? You'll need to add it in a couple places. Search for every instance in this file referencing `ColorValue`, like this: https://github.com/facebook/react-native/blob/master/packages/react-native-codegen/src/parsers/flow/components/__test_fixtures__/fixtures.js#L266-L270

Reply:
As far as I can tell, it *is* being covered in the test fixture.    The lines you referred to above don't exist anymore.   All the references in the fixtures.js file are 'ColorPrimitive' which is what the Flow type 'ProcessedColorValue' maps to.   And indeed the tests are passing.

Reply:
I linked to the tests for the Flow parser, I think you are referring to the tests for the generators. 

I wouldn't expect this change to cause tests to fail because no tests exercise the ProcessedColorValue type. 

Where do you see those lines not existing anymore? It looks like the lines linked to still exist and reference ColorValue. You'll want to duplicate those lines and change that to ProcessedColorValue

Reply:
Sorry, got it.  I was looking at:
`packages/react-native-codegen/src/generators/components/__test_fixtures__/fixtures.js`.
I just updated:
`packages/react-native-codegen/src/parsers/flow/components/__test_fixtures__/fixtures.js`
and its corresponding snapshot files.
Tests are passing